### PR TITLE
Remove remaining documentation references to the VAE deployment

### PR DESCRIPTION
**Pull request by sbesson**
Following #311 this removes mentions of the VAE deployment in a few remaining places (code + documentation) /cc @jburel

### CODE REVIEW

File: docs/operating-procedures.md [hunk 1]
```diff
@@ -94,11 +94,6 @@ possible to create a CSV output of the results:
 
     $ screen -dmS cache parallel --eta --sshloginfile nodes -a ids.txt --results /tmp/cache/ --sqlandworker csv:////%2Ftmp%2Fcache.csv -j10 '/opt/omero/server/OMERO.server/bin/omero render -s localhost -u public -w public test --force'
 
-## Analysis IDR
```
Comment (jburel) [inline docs/operating-procedures.md:97]: technically speaking it is. 
Craig forked the repo and he is using it for the new infra 
cf. https://github.com/bioimagearchive/k8s-jupyterhub

Comment (jburel) [reply docs/operating-procedures.md:97]: We can remove it from our doc but maybe add a link to new infra

Comment (joshmoore) [reply docs/operating-procedures.md:97]: @jburel : where do you want those docs? MT wiki? Or something more public?

**Comment by joshmoore** [comment]
I didn't check for _completeness_ (i.e. is there more that can go) but in terms of _correctness_ this looks fine to me. :+1:

**Comment by sbesson** [comment]
Merging and I will leave you to action https://github.com/IDR/deployment/pull/312#discussion_r571917548 @jburel

**Pull request merged**